### PR TITLE
Fix github url strings (org edx -> openedx)

### PR DESCRIPTION
**Pull request by sarina**
## This PR was autogenerated

This pr replaces the old GitHub organization, github.com/edx, with the new GitHub organization, github.com/openedx.

Tagging @openedx/tcril-engineering for review, but others are welcome to provide review.

Ref: https://github.com/openedx/tcril-engineering/issues/42

### CODE REVIEW

File: openedx_filters/filters.py [hunk 1]
```diff
@@ -55,6 +56,7 @@ def __init__(self, filter_type, running_pipeline, **extra_config):
         self.running_pipeline = running_pipeline
         self.extra_config = extra_config
 
+    @abstractmethod
```
Comment (sarina) [inline openedx_filters/filters.py:59]: I added this to get around the new pylint error `R0201: Method could be a function (no-self-use)`. This feels to me like the best solution given your docstring!

**Comment by sarina** [comment]
@mariajgrimaldi - this is ready for your review 😁

File: openedx_filters/filters.py (hunk 1 above)
Comment (mariajgrimaldi) [reply openedx_filters/filters.py:59]: Good catch! Thank you :smile:

**Review by mariajgrimaldi — APPROVED** [review]
Thanks :sun_with_face:

**Pull request merged**